treeherder/perf/alerts.py

```diff
@@ -120,3 +122,98 @@ def generate_new_alerts_in_series(signature):
                         'new_value': new_value,
                         't_value': t_value
                     })
+
+
+@dataclass
+class AlertSelectionConfig():
+    max_alerts_selected = 5
+    platforms_of_interest = ('windows10', 'windows7', 'linux', 'osx', 'android')
+
+
+def os_relevance(alert_platform: str):
+    platforms = [
+        {
+            'name': 'windows10',
+            'relevance': 5
+        },
+        {
+            'name': 'windows7',
+            'relevance': 4
+        },
+        {
+            'name': 'linux',
+            'relevance': 3
+        },
+        {
+            'name': 'osx',
+            'relevance': 2
+        },
+        {
+            'name': 'android',
+            'relevance': 1
+        }
+    ]
+
+    for platform in platforms:
+        if platform['name'] in alert_platform:
+            return platform['relevance']
+    return 0
+
+
+def ensure_platform_variety(sorted_all_alerts: list, alert_selection_config=AlertSelectionConfig):
+    # Note: Ensure that the sorted_all_alerts container has only
```

Comment:
It would be better to replace all function `#` comments with tripple-quotes docstrings.

Reply:
Replaced with canonical docstrings.